src/pages/MyCoverCropList/MyCoverCropComparison/MyCoverCropComparison.js

```diff
@@ -260,9 +256,9 @@ const MyCoverCropComparison = ({ selectedCrops }) => {
         </div>
         {/* Actual crops show up from here */}
```

Comment:
Hi @Adamws33 Is this the ternary you were mentioning.

Reply:
Yes line 254/250

Reply:
Right, got it and should I make any change or have you handled this one on your side.